__tests__/articles.test.js

```diff
@@ -43,6 +36,14 @@ describe("/api/articles/:article_id", () => {
         expect(body.article).toHaveProperty("created_at");
         expect(body.article).toHaveProperty("votes");
         expect(body.article).toHaveProperty("article_img_url");
+        expect(typeof body.article.author).toBe("string");
+        expect(typeof body.article.title).toBe("string");
+        expect(typeof body.article.article_id).toBe("number");
+        expect(typeof body.article.body).toBe("string");
+        expect(typeof body.article.topic).toBe("string");
+        expect(new Date(body.article.created_at)).toBeInstanceOf(Date);
```

Comment:
because you're invoking this with the new keyword it will always be an instance of Date

Reply:
@Hard-Luck  Thank you!  I wasn't sure if the Date constructor would throw an error if passed something that couldn't be turned into a date.  Would there be a better way to deal with this check?